test/common/export_results.c

```diff
@@ -68,23 +70,22 @@ int test_common_parse_options(int argc, char *argv[])
 	env = getenv("TEST_COMMON_EXPORT");
 	if (env) {
 		gbl_data.common_options.is_export = true;
-		odph_strcpy(gbl_data.filename, env, MAX_FILENAME_LEN);
+		odph_strcpy(gbl_data.filename, env, MAX_PATH_LEN);
 	}
 
 	/* Find and remove option */
 	for (i = 0; i < argc;) {
 		if (strcmp(argv[i], "--test-common-export") == 0) {
 			gbl_data.common_options.is_export = true;
+			odph_strcpy(gbl_data.command, argv[0], MAX_PATH_LEN);
```

Comment:
As this is always from `argv[0]` could be done inside the `if (gbl_data.common_options.is_export) { }` below.